### PR TITLE
move message logic up to module

### PR DESCRIPTION
**Pull request by mockdeep**

### CODE REVIEW

**Comment by mockdeep** [comment]
Wow, subtle difference on this one. These specs were passing locally, but because of the time zone difference it changes the regex. Locally, it has `-0800`, which is just characters in a regex, but on TravisCI it has `+0000`, where the `+` is a special character.

**Comment by mockdeep** [comment]
Fixed!

File: lib/stink_bomb.rb [hunk 1]
```diff
@@ -27,17 +28,25 @@ def reset
     def bombs
       configuration.bombs
     end
-  end
 
-private
+  private
+
+    def sub_message(deadline, message)
```
Comment (golmansax) [inline lib/stink_bomb.rb]: maybe `fill_in_message`?  I was confused about `sub` until I saw the implementation.

**Comment by mockdeep** [comment]
renamed to `fill_in_message`

**Pull request merged**